Android/FirebaseJobDispatcher/samples/Firebase.JobDispatcher.TestApp/DemoJob.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Android.App;
+using Android.Util;
+using Firebase.JobDispatcher;
+
+namespace FJDTestApp
+{
+    [Service(Name = "com.xamarin.fjdtestapp.DemoJob")]
+    [IntentFilter(new[] {FirebaseJobServiceIntent.Action})]
+    public class DemoJob : JobService
+    {
+        static readonly string TAG = "X:DemoService";
+
+        public override bool OnStartJob(IJobParameters p0)
```

Comment:
Good catch. It seems that I forgot to include the commit with an updated `MetaData.xml` that has the parameters named.